Announcement about the new GRASS PSC (#478)
OSGeo/grass-website: content/news/2024_10_11_refreshing_grass_psc.md

---
title: "Results of GRASS GIS PSC elections"
date: 2024-10-11T11:13:00+02:00
layout: "news"
author: GRASS Development Team
---

### Welcome to the Newly Elected Members of the GRASS GIS PSC!

In mid-September, the GRASS GIS community began the election process 
to fill the seats previously held by **Helena Mitasova**, **Helmut Kudrnovsky**, 
**Martin Landa**, and **Michael Barton**. After nominations from the community, 

> *10 GRASS GIS contributors* stood as candidates for these four PSC positions.

Reviewer: While specific contributions and community involvement is needed to vote, we purposefully have no such requirements for PSC candidates to allow any member of the community to be part of PSC. Let's use community members here not to create false impression about the general requirements.

```suggestion
*10 GRASS GIS community members* stood as candidates for these four PSC positions.
```

After the election itself, the GRASS GIS PSC welcomed new members, 
namely **Linda Karlovska**, **Nicklas Larsson**, and **Paulo van Breugel**, 
while **Helmut Kudrnovsky** was re-elected for another term. 
The GRASS PSC is now composed of the following nine members:

* Verónica Andreo (continuing member)
* Huidae Cho (continuing member)
* Markus Neteler (continuing member)
* Vaclav Petras (continuing member)
* Anna Petrášová (continuing member)
* Linda Karlovská (new member, 41 votes)  
* Helmut Kudrnovsky (re-elected member, 30 votes)  
* Nicklas Larsson (new member, 30 votes)
* Paulo van Breugel (new member, 29 votes)

> For completeness, election resuts and
> details about the voting process, are available on the

Reviewer 2: ```suggestion
For completeness, the election results and the
details about the voting process are available on the
```

---
title: "Results of GRASS GIS PSC elections"
date: 2024-10-11T11:13:00+02:00
layout: "news"
author: Linda Karlovská
---

### Welcome to the Newly Elected Members of the GRASS GIS PSC!

> By mid-September, the GRASS GIS community began the election process 
> to replace four of the nine Project Steering Committee (PSC) seats .
> These were previously held by **Helena Mitasova**, **Helmut Kudrnovsky**, 
> **Martin Landa**, and **Michael Barton**. After nominations from the community, 
> *10 GRASS GIS community members* stood as candidates for the positions.

Reviewer 2: ```suggestion
In mid-September, the GRASS GIS community began the election process
to replace four of the nine seats on the Project Steering Committee (PSC).
These seats were previously held by **Helena Mitasova**, **Helmut Kudrnovsky**, 
**Martin Landa**, and **Michael Barton**. Following nominations from the community,
*10 members of the GRASS GIS community* ran for these positions.
```
